src/main/java/com/iota/iri/cache/CacheConfiguration.java

```diff
@@ -0,0 +1,128 @@
+package com.iota.iri.cache;
+
+/**
+ * Cache configuration. This class represents the config values used to instantiate a {@link Cache}
+ */
+public class CacheConfiguration {
```

Comment:
Let's make this an interface that exposes getters only.
This will allow us to have caches where no one will start playing with the setters.

It might also work better with our DI later on